src/components/shared/Subscribe.vue

```diff
@@ -74,29 +74,31 @@ export default {
     :class="{ fixed: fixed }"
     class="subscribe-widget"
   >
-    <span>Güncel iş ilanlarını</span>
-    <div class="subscribe--select">
-      <multiselect
-        v-model="frequencyModel"
-        :options="frequencyOptions"
-        label="text"
-        :searchable="false"
-        :close-on-select="true"
-      />
-    </div>
-    <span>olarak almak için</span>
-    <input type="text" v-model="name" placeholder="İsim soyisim" class="username">
-    <input type="text" v-model="email" placeholder="Email">
-    <button class="button" @click="handleSubscribe">
-      Abone ol!
-    </button>
-    <span
-      v-if="fixed"
-      class="close"
-      @click="close"
-    >
-      Bir daha gösterme :(
-    </span>
+    <form @submit.prevent="handleSubscribe">
```

Comment:
We don't need to have a form here because we have a button with `handleSubscribe` click handler on line 91.

Right now we have a button and a form which call the same handler function. Can we remove the form?

Reply:
I added the form to allow user to submit form by pressing `Enter` after inputting the email, as well as a mouse click. Actually, a better solution is to add `v-on:keyup.enter` to the last input.

If this is not necessary, I can delete it.

Reply:
Oh, that makes very sense 👍

My only concern is, since we now have the form submit and button click hooked up to the same handler, if I click to the button to subscribe, will the handler triggered twice or not? If you can confirm the handler will only be triggered once, I am ok with this implementation.

Thank you for thinking this case.

Reply:
I didn't think about double submit... So, I changed once again.

Instead of a form, I used ` v-on:keyup.enter` on the last input with a `handleEnter()` handler, which in turn calls `handleSubscribe()`. The subscription handler is called only once, even if you click on the button and press Enter at the same time.

Reply:
But I believe, if you hit enter multiple times, the request will be made multiple times. I don't see a code to prevent it.

> The subscription handler is called only once, even if you click on the button and press Enter at the same time.

I also don't understand how this could be possible. Which part of the code prevents this?

Reply:
You are right, the request was being sent multiple times, I didn't realize that. 
I changed the code so that as soon as the enter key is hit/ button is clicked, the subscribe panel's `isClosed` is set to true, and in the catch block (if there is an error, the user should be able to edit the details) `isClosed` is set to false again.